gitter/messages.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import requests
+
+from data.webservices import webservices_url
+from gitter.score import sentiment_score
+from gitter.models import Question, Answer
+
+
+def get_messages():
+    """
+    Get all the messages send by newcomers on the gitter rooms.
+    """
+    logger = logging.getLogger(__name__)
+    IMPORT_URL = webservices_url('messages')
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=IMPORT_URL,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
+
+    data = response.json()
+    return data
+
+
+def message_score(message):
+    """
+    Get the score for a message dict.
+
+    :param message: a message dict of type:
+                    {
+                       "identifier": "5b588269c0fa8016e7379191",
+                       "room": "offtopic",
+                       "sent_at": "2018-07-25 14:00:09.456000+00:00",
+                       "sent_by": "Naveenaidu",
+                        "text": "How can I solve this issue?"
+                    },
+    :return: an integer reperesenting score of the message text,
+             if it's positive the message should be ignored, if it's
+             negative the message should be saved into Question model,
+             and if it's zero, the message should be saved into Answer model.
+
+    """
+    text = message['text']
+    return (sentiment_score(text))
+
+
+def import_messages(message):
+    logger = logging.getLogger(__name__)
+    sent_by = message.get('sent_by')
+    score = message_score(message)
```

Comment:
no use of word 'score' anywhere in the main app logic.  The nlp layer should spit out a 'message classification'.

maybe create an `enum` `MessageCategory` / `MessageType` or something like that.